content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -235,36 +318,45 @@ When you use `ILIKE`, the `%` wildcard character(s) should be included in the va
 ```ts
 const userName = 'Sarah'
 const emailFragment = 'prisma.io'
-const result = await prisma.$queryRaw(
-  'SELECT * FROM "User" WHERE (name = $1 OR email = $2)', // Using %$2 here would not work
+const result = await prisma.$queryRaw`
+  SELECT * FROM "User" WHERE (name = $1 OR email = $2), 
   userName,
-  `%${emailFragment}`
-)
+  %${emailFragment}`
 ```
 
+> **Note**: Using %$2 as an argument would not work
+
 #### <inlinecode>ALTER</inlinecode> limitation (PostgreSQL)
 
 PostgreSQL [does not support using `ALTER` in a prepared statement](https://www.postgresql.org/docs/current/sql-prepare.html), which means that the following queries will not work:
 
 ```ts
 await prisma.$executeRaw`ALTER USER prisma WITH PASSWORD '${password}'` // Tagged template
-await prisma.$executeRaw(`ALTER USER prisma WITH PASSWORD $1`, newPassword) // Parameterized query
+await prisma.$executeRaw`ALTER USER prisma WITH PASSWORD $1, newPassword` // Parameterized query
 ```
 
 You can use the following query to [work around the issue](https://github.com/prisma/prisma-client-js/issues/940#issuecomment-748014175), but be aware that this is potentially **unsafe** as `${password}` is not escaped:
 
 ```ts
-await prisma.$executeRaw(`ALTER USER prisma WITH PASSWORD '${password}'`)
+await prisma.$executeRaw`ALTER USER prisma WITH PASSWORD '${password}'`
 ```
 
 ## SQL injection
 
 Prisma Client mitigates the risk of SQL injection in the following ways:
 
-- Prisma Client escapes all variables when you use tagged templates (` $queryRaw`` ` or ` $executeRaw`` ` ) or parameterized queries (`$queryRaw(query, variables)` or `$executeRaw(query, variables)`), and sends all queries as prepared statements.
-- Both versions of `$executeRaw` (standard and tagged template) can only run **one** query at a time. You cannot append a second query - for example, adding `DROP bobby_tables` to the end of an `ALTER`.
+- Prisma Client escapes all variables when you use tagged templates or parameterized queries and sends all queries as prepared statements.
 
-If you cannot use tagged templates or parameterized queries (for example, you need to pass in table or column names as variables), be aware that your code may be vulnerable to SQL injection.
+  ```ts
+  $queryRaw`` // Tagged template
+  $executeRaw`` // Tagged template
+  $queryRaw`query, variables` // Parameterized query
+  $executeRaw`query, variables` // Parameterized query
+  ```
+
+- Both versions of `$executeRaw` (tagged template and parameterized) can only run **one** query at a time. You cannot append a second query - for example, adding `DROP bobby_tables` to the end of an `ALTER`.
+
+If you cannot use tagged templates or parameterized queries (for example, you need to pass in table or column names as variables), you can instead use [`$queryRawUnsafe`](#query-raw-unsafe) or [`$executeRawUnsafe`](#execute-raw-unsafe) but **be aware that your code may be vulnerable to SQL injection**.
```

Comment:
Note for later: it could be good to point people to how to properly escape strings etc. to prevent SQL injections whenever using Unsafe signature methods. This can be done in a separate effort, but would be worthwhile as a user, since you otherwise just see you're in this situation but left with the task to figure out how to deal with it on your own.

Reply:
I agree, a new section within this page about how to escape strings while using `$executeRawUnsafe` / `$queryRawUnsafe` would be ideal. Ive asked @millsp about that. Would be good to link to it from within the page

Reply:
Somewhat related: I'm wondering if we could allow `ALTER`, `CREATE`, `DROP` in `$executeRaw` and instead of doing a prepared statement, we would automatically escape their inputs. I also think that we should look into providing some helper to escape input.

Reply:
@millsp is this something you are thinking about for 7th sept release?

Reply:
Those are interesting ideas indeed @millsp and I think we should track them in their individual issues, unless they would come with a breaking change, in such case that would be better to indeed do those before the new major.

Reply:
That won't be a breaking change, so I'll keep the improvements for later.